src/components/group.js

```diff
@@ -49,6 +49,7 @@ function Group(widget, container, groupSpec, options, index) {
 
 	this._canDrag = false;
 	this._dragging = false;
+	this._collapsible = groupSpec.collapsible !== undefined ? groupSpec.collapsible : true;
```

Comment:
It seems there's a `Group.prototype.replace` method which updates the group spec and I guess the collapsible state should be updated in there as well.